<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentView</class>
 <widget class="QWidget" name="StudentView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>个人信息：</string>
   </property>
  </widget>
  <widget class="QLabel" name="StuInfo">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>411</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="StuInfoBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>个人信息查看</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PasswordBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>400</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>更改密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordlineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>400</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入更改后的密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
